.gitignore

```diff
@@ -0,0 +1,5 @@
+# Rakudo compilation cache
+.precomp
+
+# Vim temporary file
+*.swp
```

Comment:
Not a blocker, but `*.swp` is author-specific. I'd rather not put that in a per-project .gitignore. I recommend adding it to `~/.gitignore`. I prefer having `*.sw?` myself because sometimes vim will outdo itself in creating .swp, then swo, and then .swn... if things get really screwy.

Reply:
Thanks,
That is good pointer, I'm still pretty new to vim so I've only accidentally grabbed .swp files so far and have been just ignoring them in my project files. That's a much better solution.